### PR TITLE
mixxx.exe copyright fix

### PR DESCRIPTION
**Pull request by daschuer**
I have noticed a mismatch between our copyright, and what is shown in the "Details" view when right click on mixxx.exe.

Taken Microsoft Products as template I have removed the preceding Copyright, because this is already displayed before the statement in the table column.

### CODE REVIEW

File: vamp-plugins/maths/Polyfit.h [hunk 1]
```diff
@@ -31,7 +31,7 @@
              specific number of points.
 2001 Jan 07  Update Web site address
 
-Copyright � David J Taylor, Edinburgh and others listed above
```
Comment (Holzhaus) [inline vamp-plugins/maths/Polyfit.h:34]: Why did you replace this? This looks unrelated.

File: src/mixxx.rc [hunk 2]
```diff
@@ -8,8 +8,9 @@ IDI_ICON1               ICON    DISCARDABLE     "res/images/ic_mixxx.ico"
 #define VER_PRODUCTNAME_STR         "Mixxx\0"
 #define VER_FILEDESCRIPTION_STR     "Mixxx digital DJ software"
 #define VER_COMPANYNAME_STR         "The Mixxx Development Team"
-#define VER_LEGALCOPYRIGHT_STR      "Copyright �2001-" CUR_YEAR " The Mixxx Development Team and/or its subsidiary(-ies).\0"
-#define VER_ORIGINALFILENAME_STR         "Mixxx.exe"
+// \xA9 for (c) symbol. The Microsoft Resource Compiler compiler asumes Latin-1 strings
+#define VER_LEGALCOPYRIGHT_STR      "\xA9 2001-" CUR_YEAR " Mixxx Development Team\0"
```
Comment (Holzhaus) [inline src/mixxx.rc:12]: `and/or its subsidiary(-ies).` ?

**Review by Holzhaus — COMMENTED** [review]
Thanks!

File: vamp-plugins/maths/Polyfit.h (hunk 1 above)
Comment (daschuer) [reply vamp-plugins/maths/Polyfit.h:34]: Because it was a latin-1 encoded (c), It looks like � in eclipse. 
We should just avoid non ascii character in our source.

File: src/mixxx.rc (hunk 2 above)
Comment (daschuer) [reply src/mixxx.rc:12]: Removed because we have non. We have this appendix no where else.

File: vamp-plugins/maths/Polyfit.h (hunk 1 above)
Comment (uklotzde) [reply vamp-plugins/maths/Polyfit.h:34]: Do we really need to edit external files without any real need?

Comment (daschuer) [reply vamp-plugins/maths/Polyfit.h:34]: I have found it via a lind tool during packaging. This file is no longer extern in 2.3. but if you all prefer to revert it I can do. Should I?

Comment (uklotzde) [reply vamp-plugins/maths/Polyfit.h:34]: Let's revert those changes.

**Comment by uklotzde** [comment]
Since 2.2.4 has already been tagged should we re-target this to 2.3?

**Comment by daschuer** [comment]
Done.

**Comment by uklotzde** [comment]
LGTM
